dedibox-console/my-account/how-to/accept-outsourcing.mdx

```diff
@@ -7,32 +7,30 @@ content:
   paragraph: This page explains how to accept an outsourcing request.
 tags: dedibox outsource outsourcing-request
 dates:
-  validation: 2023-01-16
+  validation: 2023-07-24
   posted: 2020-03-12
 categories:
   - account
 ---
 
-If you have a Scaleway Dedibox account, another Dedibox user can give you access to their Dedibox dedicated servers and failover IPs by outsourcing them to you. Outsourcing allows you to manage the outsourced services without full access to another user's Dedibox account.
+If you have a Scaleway Dedibox account, another user can grant you access to their dedicated servers and failover IPs by outsourcing them to you. This outsourcing process allows you to manage the services without gaining full access to the other user's Dedibox account.
 
 <Message type="requirement">
-  - You have an account and are logged into the [Dedibox Console](https://console.online.net) 
+  - You have an account and are logged into the [Dedibox console](https://console.online.net) 
 </Message>
 
-Once the other user [sends you an outsourcing request](/dedibox-console/my-account/how-to/outsource-dedibox), you receive a notification to accept the incoming request.
+Follow these steps to accept an outsourcing request:
 
-1. Click the link in the outsourcing request email to open the Scaleway Dedibox console.
+1. When another user [sends you an outsourcing request](/dedibox-console/my-account/how-to/outsource-dedibox), you will receive a notification. Click the provided link in the outsourcing request email to open the Scaleway Dedibox console.
     <Lightbox src="scaleway-dedibox-console-outsourcing-request.webp" width="600" alt="" />
-
-    The **Outsourcing** section of the Scaleway Dedibox console displays.
-2. Check the **Accept** box for the service(s) you agree to have outsourced to you.
+2. The **Outsourcing** section of the Scaleway Dedibox console displays.
+4. Review the services that the user wants to outsource to you.
+5. Check the **Accept** box for the service(s) that you agree to take over.
     <Lightbox src="scaleway-dedibox-console-outsourcing-accept.webp"width="600" alt="" />
-3. Click **Save changes** to validate your choice.
-
-The services are now outsourced and available in your console.
-
+6. Click **Save changes** to confirm your acceptance.
```

Comment:
```suggestion
6. Click **Save changes** to confirm.
```

Reply:
I think we should stay consistent and only use "Dedibox console", I have seen some other "Dedibox Console"s throughout the review